include/boost/multiprecision/jacobi.hpp

```diff
@@ -0,0 +1,100 @@
+//---------------------------------------------------------------------------//
+// Copyright (c) 2018-2020 Mikhail Komarov <nemo@nil.foundation>
+//
+// Distributed under the Boost Software License, Version 1.0
+// See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt
+//---------------------------------------------------------------------------//
+
+#ifndef BOOST_MULTIPRECISION_JACOBI_HPP
+#define BOOST_MULTIPRECISION_JACOBI_HPP
+
+#include <boost/multiprecision/detail/default_ops.hpp>
+
+namespace boost {
+namespace multiprecision {
+
+template <typename Backend>
+inline int eval_jacobi(const Backend& a, const Backend& n)
+{
+   using default_ops::eval_get_sign;
+   using default_ops::eval_gt;
+   using default_ops::eval_integer_modulus;
+   using default_ops::eval_is_zero;
+   using default_ops::eval_lsb;
+   using default_ops::eval_lt;
+   using default_ops::eval_modulus;
```

Comment:
I need eval_divide in this list as well to compile.

Reply:
Agreed. Missed it during review. Fixed.